make/toolchain-android-aarch64.cmake

```diff
@@ -0,0 +1,97 @@
+# **********************************************************
+# Copyright (c) 2014-2016 Google, Inc.    All rights reserved.
+# **********************************************************
+
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice,
+#   this list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of Google, Inc. nor the names of its contributors may be
+#   used to endorse or promote products derived from this software without
+#   specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL GOOGLE, INC. OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+# DAMAGE.
+
+# For cross compiling for 64-bit arm Android using the Android LLVM toolchain:
+# - Download toolchain, and install the standalone toolchain
+#   https://developer.android.com/ndk/downloads/revision_history
+#   $/PATH/TO/ANDROID_NDK/toolchains/llvm/prebuilt/<build>/bin
+# - Build ZLIB with the Android toolchain (included in third_party/zlib)
+#   $AR=/TOOLCHAIN/INSTALL/PATH/llvm-ar \
+#   CC=/TOOLCHAIN/INSTALL/PATH/aarch64-linux-android<chosen-version>-clang \
+#   CFLAGS="-fPIC -O" ./configure --static && make install prefix=$HOME/zlib
+# - cross-compiling config with ANDROID_TOOLCHAIN
+#   $cmake -DCMAKE_TOOLCHAIN_FILE=../dynamorio/make/toolchain-android-aarch64.cmake \
+#     -DANDROID_TOOLCHAIN=/TOOLCHAIN/INSTALL/PATH -DTOOLCHAIN_VERSION=<chosen-version> \
+#     -DZLIB_LIBRARY=$HOME/zlib/lib/libz.a -DZLIB_INCLUDE_DIR=$HOME/zlib/include \
+#     ../dynamorio
+
+# Target system
+set(CMAKE_SYSTEM_NAME Android)
+set(CMAKE_SYSTEM_VERSION 1)
+
+# If using a different target, set -DTARGET_ABI=<abi> on the command line.
+if (NOT DEFINED TARGET_ABI)
+  set(TARGET_ABI "aarch64-linux-android")
+endif ()
+if (TARGET_ABI MATCHES "^aarch64")
+  set(CMAKE_SYSTEM_PROCESSOR aarch64)
+endif ()
+
+# specify the cross compiler
```

Comment:
style: Capitalize and punctuate comments in new code.  Ditto below.

Reply:
Done.